packages/koa-shopify-auth/src/auth/client/request-storage-access.ts

```diff
@@ -1,12 +1,12 @@
 // Copied from https://github.com/Shopify/shopify_app
-const requestStorageAccess = (shop: string) => {
+const requestStorageAccess = (shop: string, prefix = '') => {
   return `(function() {
       function redirect() {
         var targetInfo = {
           myshopifyUrl: "https://${shop}",
-          hasStorageAccessUrl: "/auth/inline?shop=${shop}",
-          doesNotHaveStorageAccessUrl: "/auth/enable_cookies?shop=${shop}",
-          appTargetUrl: "/?shop=${shop}"
+          hasStorageAccessUrl: "${prefix}/auth/inline?shop=${shop}",
+          doesNotHaveStorageAccessUrl: "${prefix}/auth/enable_cookies?shop=${shop}",
+          appTargetUrl: "${prefix}/?shop=${shop}"
```

Comment:
I don't think we should set `prefix` here. The path that the application lives on is orthogonal to the paths that the OAuth logic lives on. (You can tell this because the `verifyRequest` middleware can be applied separately from the `shopifyAuth` middleware.)

In a separate PR, however, I'd be open to a couple alternatives:
1. A new `applicationUrl` configuration option that would specify the path to direct the user when cookie access is enabled.
2. Logic to pass a `returnTo` value through query params. (We do this in `shopify_app`, though it's pretty complex IIRC.)

Reply:
Thanks for the feedback; I'll update the PR shortly

Reply:
Ok, appTargetUrl prefix undone.